blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md

```diff
@@ -0,0 +1,143 @@
+---
+title: How to Store NFT Smart Contract Assets Using The FileBase Decentralized Storage System
+description: This article gives a comprehensive overview of decentralized storage solutions. It contains a clear walk-through of how to store digital assets on the filebase storage solution.
+author:
+  - name: John Fawole
+    title: Technical Writer
+    url: https://www.linkedin.com/in/johnfawole/
+    image_url: https://github.com/johnfawole.png
+tags: ['solidity', 'smart contract', 'celosage', 'celo', 'filebase', 'decentralized storage']
+hide_table_of_contents: true
+slug: /tutorials/how-to-store-NFT-smart-contract-assets-using-the-fileBase-decentralized-storage-system
+---
+
+![filebase-cover](https://user-images.githubusercontent.com/105144630/232922465-f519c6cd-9e66-41aa-879e-e81b6c342f3c.png)
```

Comment:
can you reference the header image from the user.tsx file, the one you uploaded, 
---according to the latest quidelines on the community